NEWS.txt

```diff
@@ -72,6 +72,11 @@ using the provided 'sstableupgrade' tool.
 New features
 ------------
     [The following is a placeholder, to be revised asap]
+    - CEP-37 Auto Repair is a fully automated scheduler that provides repair orchestration within Apache Cassandra. This
+    significantly reduces operational overhead by eliminating the need for operators to deploy external tools to submit
+    and manage repairs. See
+    https://cwiki.apache.org/confluence/display/CASSANDRA/CEP-37+Apache+Cassandra+Unified+Repair+Solution for more
+    details on the motivation and design
```

Comment:
:+1:  I think this is good for now.  I think after the doc site gets updated post merge, would be good to double back on this and link the doc page that was added for auto repairs.

Just a tiny nit to add a period `.` after design.